changelogs/fragments/254-add_ignore_txt.yml

```diff
@@ -0,0 +1,2 @@
+trivial:
```

Comment:
```suggestion
---
trivial:
```

The linter requires the start of document directive to be present

Reply:
@Qalthos committed, thanks